parsing/padding.go

```diff
@@ -0,0 +1,113 @@
+package parsing
+
+import (
+	"errors"
+	"github.com/filecoin-project/go-data-segment/types"
+)
+
+// Pad pads a general byte array in to FP32 chunks of bytes where the topmost bits of the most significant byte are 0
+func Pad(unpaddedData *[]byte) ([]types.FP32, error) {
```

Comment:
You don't need to take the pointer here. The slice is a tuple of (*array, len, capacity) and is usually passed by value.

Reply:
Thanks for the clarification, that is a very nice detail to know about arrays in Go!

Reply:
Fixed in 9872fe3b0c52186e745d15359c1a8ab15c0fd486